GHzDACs/Cleanup/dac.py

```diff
@@ -677,8 +677,25 @@ def _setPolarity(self, chan, invert):
 
     @inlineCallbacks
     def _checkPHOF(self, op, fifoReadings, counterValue):
-        # Determine for which PHOFs (FIFO offsets) the FIFO counter equals
-        # counterValue.
+        """ Determine correct PHOF for FIFO counter of counterValue
```

Comment:
To be completely 100% correct, I believe the standard is to not have a after the `"""` and to end this line with a period.